src/adaptors/amqp/qd_connector.c

```diff
@@ -205,21 +262,16 @@ const char *qd_connector_policy_vhost(const qd_connector_t* ct)
 }
 
 
-bool qd_connector_connect(qd_connector_t *ct)
+void qd_connector_activate(qd_connector_t *ct)
```

Comment:
```suggestion
void qd_connector_schedule_connection(qd_connector_t *ct)
```

Reply:
That doesn't make sense to me.  When I see "schedule" it implies that there's possibly a need to re-schedule in the future.  That isn't the case here.  The only thing that is re-scheduled is the connection re-connect attempt and that's handled auto-magically internal to the connector itself.

The intent of the name is convey that the connection manager is kicking off the internal connector state machine.  Once that's done the connection manager is totally out of the picture as far as scheduling the connection is concerned (IOW: the connection manager does not monitor the connection state nor does it periodically re-schedule the connector).